inspirehep/config.py

```diff
@@ -299,6 +308,7 @@
 # invenio keywords to elasticsearch fields
 SEARCH_ELASTIC_KEYWORD_MAPPING = {
     "author": ["authors.full_name"],
+    "a": ["authors.full_name"],
```

Comment:
I don't think we need these mappings? Those keywords are only used in Spires syntax (like `find a Ellis`) so they will first go through [this config file](https://github.com/inspirehep/invenio-query-parser/blob/master/invenio_query_parser/contrib/spires/config.py#L79) and be converted from `a` to `author`.

So `author` is the keyword that needs configuration.

Same is true for the other two.